app/views/users/show.html.erb

```diff
@@ -12,29 +12,44 @@
   <section>
     <h1>Bookmarked Segments</h1>
   </section>
-<section id="github_section">
-  <h2>Github:</h2>
-  <% if current_user.token.nil? %>
-    <%= "Must sign in with Github" %>
-  <% else %>
-    <h4>Repositories:</h4>
-    <% @github_info.get_repositories.each do |repo| %>
-      <section id="repo">
-       <p><%= link_to repo.name, repo.html_url %></p>
-     </section>
+  <section id="github_section">
+    <h2>Github:</h2>
+    <% if current_user.token.nil? %>
+      <%= "Must sign in with Github" %>
+    <% else %>
+      <h4>Repositories:</h4>
+      <% @github_info.get_repositories.each do |repo| %>
+        <section id="repo">
+         <li><%= link_to repo.name, repo.html_url %>
+       </section>
     <% end %>
-    <h4>Followers:</h4>
-    <% @github_info.get_followers.each do |follower| %>
+
+      <h4>Followers:</h4>
       <section id="followers">
-        <p><%= link_to follower.login, follower.html_url %></p>
+        <% @github_info.get_followers.each do |follower| %>
+          <li><%= link_to follower.login, follower.html_url %>
+          <% if follower.find_by_id %>
+          <%=  button_to "Add as Friend", controller: "friendships", action: "create", method: "post", params: {github_id: follower.github_id} %>
+          <% end %>
+        <% end %>
       </section>
-    <% end %>
-    <h4>Following:</h4>
-    <% @github_info.get_following.each do |following| %>
+
+      <h4>Following:</h4>
       <section id="following">
-        <p><%= link_to following.login, following.html_url %></p>
+        <% @github_info.get_following.each do |following| %>
+        <li><%= link_to following.login, following.html_url %>
+          <% if following.find_by_id %>
+          <%=  button_to "Add as Friend", controller: "friendships", action: "create", method: "post", params: {github_id: following.github_id} %>
+          <% end %>
+        <% end %>
       </section>
-    <% end %>
-    <% end %>
+
+      <h4>Friends:</h4>
+      <section id='friends'>
+        <% current_user.friends.each do |friend| %>
+          <%= "#{friend.first_name} #{friend.last_name}" %>
```

Comment:
List all the friends in this section of the current_user so it will work for whoever signs in to the app and not just one user